main.go

```diff
@@ -512,6 +563,10 @@ func FlashGDB(pkgName, target, port string, ocdOutput bool, config *BuildConfig)
 				// Maybe we should send a .Kill() after x seconds?
 				daemon.Wait()
 			}()
+		case "msd":
+			return errors.New("gdb is not supported for drag-and-drop programmable devices")
```

Comment:
It is possible to get GDB to work with boards that use MSD flashing such as the SAMD21 but you need to have an external JLink adaptor and some way to connect to the pins, which might require additional specialized connectors.

See https://learn.adafruit.com/debugging-the-samd21-with-gdb/setup for details.

In any case, that is probably better addressed in a separate future PR, as it currently requires extra steps beyond the `tinygo flashgdb` command.

Reply:
What I did was that if `msd` is selected, that I will check whether there is an OpenOCD configuration available anyway and switch to that if available. So for example, with this PR the micro:bit is flashed using msd but for `tinygo gdb` it will use the openocd configuration that's also available. This gives the best of both worlds: no external tool is needed for simply flashing, and `tinygo gdb` still works.

(This also happens to be part of the motivation behind this PR: I wanted to preserve `tinygo gdb` support because I find it very useful sometimes).